sdk/search/azure-search/tests/async_tests/test_live_async.py

```diff
@@ -118,6 +118,36 @@ class SearchIndexClientTestAsync(AzureMgmtTestCase):
             assert all(set(x) == expected for x in results)
             assert all(x['category'] == "Budget" for x in results)
 
+    @ResourceGroupPreparer(random_name_enabled=True)
+    @SearchServicePreparer(schema=SCHEMA, index_batch=BATCH)
+    @await_prepared_test
+    async def test_get_search_facets_none(self, api_key, endpoint, index_name, **kwargs):
+        client = SearchIndexClient(
+            endpoint, index_name, SearchApiKeyCredential(api_key)
+        )
+
+        query = SearchQuery(search_text="WiFi")
+        query.select("hotelName", "category", "description")
+
+        async with client:
+            results = await client.search(query=query)
+            assert await results.facets is None
```

Comment:
@bryevdv Is it possible to access other properties of the result like `coverage` and `totalCount` in the same way as `facets`?

Reply:
@brjohnstmsft it could be, but isn't. Give EOD deadline and the interaction of coverage with minimumCoverage, can an issue for that be added and put on preview 2, rather than rushing in today?

Reply:
Sure -- I'm assuming adding new properties/methods is a non-breaking change anyway. Just making sure the design allows for the addition of new stuff on the response.

Reply:
Yup no issue adding later. Here is the issue https://github.com/Azure/azure-sdk-for-python/issues/10187